CoronaCare/src/screens/CameraScreen.js

```diff
@@ -0,0 +1,66 @@
+import React, { PureComponent } from 'react';
+import { AppRegistry, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import { RNCamera } from 'react-native-camera';
+
+export default class CameraScreen extends PureComponent {
```

Comment:
Can you find out why must it be a PureComponent. Maybe can try using Component and see if it works